lib/exhibit.rb

```diff
@@ -0,0 +1,10 @@
+class Exhibit
+
+  attr_reader :exhibit_name, :exhibit_cost
+  def initialize(exhibit_info)
+    @exhibit_name = exhibit_info[:name]
+    @exhibit_cost = exhibit_info[:cost]
+  end
```

Comment:
We are already in the Exhibit class so we should use only `name`, `cost`. Since you changed what the variables were called tests failed on my end

```suggestion
  attr_reader :name, :cost
  def initialize(exhibit_info)
    @name = exhibit_info[:name]
    @cost = exhibit_info[:cost]
  end
```